lib/hammer_cli_foreman_remote_execution/remote_execution_feature.rb

```diff
@@ -0,0 +1,40 @@
+module HammerCLIForemanRemoteExecution
+  class RemoteExecutionFeature < HammerCLIForeman::Command
+    resource :remote_execution_features
+    desc _('Manage remote execution features')
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _('ID')
+        field :name, _('Name')
+        field :description, _('Description')
+        field :job_template_name, _('Job template name')
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output do
+        field :id, _('ID')
+        field :label, _('Label')
+        field :name, _('Name')
+        field :description, _('Description')
+        field :job_template_name, _('Job template name')
```

Comment:
Also, I think it'd be nice if we included the job template id in the details.  It makes it easier if I want to quickly view the template, I can do `job-template info --id XX` instead of cutting and pasting the name, quoting it, etc.